src/migration/1640288411208-route-slug.ts

```diff
@@ -0,0 +1,46 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class routeSlug1640288411208 implements MigrationInterface {
+  name = 'routeSlug1640288411208';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "route" ADD "slug" character varying`);
+
+    await queryRunner.query(
+      `update route set slug = btrim(regexp_replace(regexp_replace(unaccent(lower(name)), '[\''\(\)\ \,\.\+\#\?\!\`]', '-', 'g'), '-+', '-', 'g'), '-')`,
+    );
+
+    let n = 1;
+
+    while (await this.hasDupes(queryRunner)) {
+      await queryRunner.query(`
+            update route set slug = concat(slug, '--${n}') 
```

Comment:
Is there a reason for double dash before the number or could we use single dash to be consistent with crag an club slugs?

Reply:
You are right, single will be more consistent. Fixed.